cookbook/attributes/default.rb

```diff
@@ -14,21 +14,8 @@
   },
 }
 
-default[:mesos] = {
-  :package_version => "0.21.0-1.0.ubuntu1404",
-  :common => {
-    :ip => private_ip,
-  },
-  :master => {
-    :cluster => 'vagrant-singularity',
-  },
-  :slave => {
-    :containerizers => 'docker,mesos',
-    :switch_user => 'false'
```

Comment:
we need to make sure these defaults get carried over to the `cookbook-mesos` version. the Vagrantfile might have to be updated too -- it sets some default slave resources